lldb/scripts/dpu/dpu_commands.py

```diff
@@ -319,6 +312,17 @@ def dpu_attach(debugger, command, result, internal_dict):
         print("Could not create dpu target")
         return None
 
+    storage = target_dpu.FindFirstGlobalVariable("error_storage")
+    if storage.IsValid():
+        lldb_server_dpu_env["UPMEM_LLDB_ERROR_STORE_ADDR"] = str(storage.location)
```

Comment:
I think it could be really important to add a comment around here to explain in details why we do like this. 
This will definitely be something we may worry about in the future when coming back again on this piece of code, and at that time it will be nice to know why an unusual behavior as passing an address through env was the right solution. ^^